runtime/runtime_options.go

```diff
@@ -34,6 +34,10 @@ const (
 	defaultTickSeriesBatchSize                  = 512
 	defaultTickPerSeriesSleepDuration           = 100 * time.Microsecond
 	defaultTickMinimumInterval                  = time.Minute
+	// defaultMaxWiredBlocks is 2MM by default which if using 2hr block sizes
+	// and writing every 10s at 1.4 point/byte (m3tsz) should use roughly 4gb:
+	// 1.4 * 6 * 120 * (2^21) = ~2gb
+	defaultMaxWiredBlocks = uint(1 << 31)
```

Comment:
Let's set this to ~256k (2^18) for now perhaps, this initial 2MM number was picked when wiredlist still included mutable blocks (that were skipped over when eviction was occurred).

Reply:
not sure why I had this at 1 << 31, that was way too high. 1 << 18 ~~ 256k right?